components/automate-cli/pkg/reportgenerator/usagegenerator.go

```diff
@@ -146,7 +162,7 @@ func queryElasticSearchNodeCount(client *elastic.Client, startTime time.Time, en
 		}
 	}
 	writer.Flush()
-	fmt.Println("The node count report is available in: ", filename)
+	fmt.Println("The node count report is available in: ", fileName)
```

Comment:
```suggestion
	logrus.Info("The node count report is available in: ", fileName)
```

Reply:
We cannot log to a file and use log rus @daveaugustus